assets/scripts/streets/image.js

```diff
@@ -45,3 +49,64 @@ export function getStreetImage (street, transparentSky, segmentNamesAndWidths, s
 
   return el
 }
+
+// Save thumbnail if necessary every 5 minutes (300000 ms)
+const SAVE_THUMBNAIL_TIME_INTERVAL = 300000
+let _lastSavedTimestamp
+let unsavedThumbnail = true
```

Comment:
Ah oops. I think I meant to have it underscored. Yes, I think I would rather have it stay in this module then to put it in the Redux store since only functions within this file would manipulate it.